src/data/items.ts

```diff
@@ -102,4 +102,10 @@ export const itemData: ItemInterface = {
 		tags: ['medical', 'cloth'],
 		description: 'Woven strip of material with absorbent barrier to prevent adhering to wounds.',
 	},
+	
+	'lindenportrait': {
+		label: 'Linden\'s Portrait',
+		tags: ['decoration', 'photo', 'furniture', 'australian'],
```

Comment:
Tags are listed [here](https://github.com/antond15/items/blob/main/src/data/tags.ts) so please use the available ones or create some new.
I would prefer using the `other` tag for this one.

Reply:
gotcha